web/services/store/index.js

```diff
@@ -0,0 +1,50 @@
+import { createStore, combineReducers, compose } from 'redux';
+import { reactReduxFirebase, firebaseReducer } from 'react-redux-firebase';
+import firebase from 'firebase';
+
+const prodConfig = {
+  // TODO: setup production firestore project
+};
+
+const devConfig = {
+  apiKey: 'AIzaSyCBkQHeikIsiYZ2yOHiqH_mGJKDWMDU500',
+  authDomain: 'nwhacks-2019.firebaseapp.com',
+  databaseURL: 'https://nwhacks-2019.firebaseio.com',
+  projectId: 'nwhacks-2019',
+  storageBucket: 'nwhacks-2019.appspot.com',
+  messagingSenderId: '98283589440',
+};
+
+const config = {
+  production: prodConfig,
+  development: devConfig,
+};
+
+// react-redux-firebase config
+const rrfConfig = {
+  // userProfile: 'users',
+  // useFirestoreForProfile: true // Firestore for Profile instead of Realtime DB
+};
+
+
+// Initialize firebase instance
+firebase.initializeApp(config[process.env.NODE_ENV]);
+
+// Add reactReduxFirebase enhancer when making store creator
+const createStoreWithFirebase =
+  compose(reactReduxFirebase(firebase, rrfConfig))(createStore);
+  // firebase instance as first argument
```

Comment:
I don't think docs like these (`firebase instance as first argument`) are very useful, especially as they accumulate, since it's explained in the library, plus a bit confusing that it appears *after* the relevant line. Better documentation in general here might be:

```js
// Add reactReduxFirebase enhancer when making store creator using our firebase instance. This does x, y, z and is used for a, b, c.
const createStoreWithFirebase = compose(reactReduxFirebase(firebase, rrfConfig))(createStore);
```

Reply:
yeahhhh i just copy pasted that from somewhere lmao